test/cli.py

```diff
@@ -295,6 +300,43 @@ def test_cli_delete(self):
             script_name = strip_dot_py(test_script)
             assert script_name not in f.getvalue()
 
+    def test_cli_print(self):
+
+        # NOTE: this is not a unit test, it relies on other command
+        # If this test is failing, make sure the following tests are passing:
+        # - test_cli_corpus
+
+        # Build the test corpus so we have builds to print
+        testargs = [
+            "benchit",
+            "benchmark",
+            "-s",
+            corpus_dir,
+            "--build-only",
+        ]
+        with patch.object(sys, "argv", testargs):
+            benchitcli()
+
+        # Make sure we can print the builds in the cache
+        for test_script in test_scripts_dot_py.keys():
+            script_name = strip_dot_py(test_script)
+            builds = filesystem.get_builds_from_script(
+                filesystem.DEFAULT_CACHE_DIR, script_name
+            )
+
+            for build_name in builds:
+                with redirect_stdout(io.StringIO()) as f:
+                    testargs = [
+                        "benchit",
+                        "cache",
+                        "print",
+                        build_name,
+                    ]
+                    with patch.object(sys, "argv", testargs):
+                        benchitcli()
+
+                    assert script_name in f.getvalue()
+
     def test_cli_version(self):
```

Comment:
Those changes probably deserve a vesion bump?

Reply:
True! Breaking ui changes. 1.0.0 if we want to start keeping track

Reply:
I'm going to leave the version number alone until the first release, which we will set to 1.0.0. Otherwise the version number will explode while we do pre-release UI tinkering (see #66, that would be another major version bump). If you object we can do a quick PR right after this one to bump the version number.